README.md

```diff
@@ -13,22 +13,36 @@ This is a fork from that code, as it was at 11/11/2013.
 The latest version of the fork can allways be found at
 https://github.com/marado/TalkerNode .
 
--------------------------------------------------------------------------
+## Installation
 
-sam says: teh SamLicence is basically "Do as you will - but if you make
-millions can I get a pint please?"
+```bash
+> git clone https://github.com/marado/TalkerNode
+> yarn # or `npm install`, if you're still in, like, two weeks ago... pffft... 
```

Comment:
# or `npm install`
we don't need the npm users dissing, specially when, until this PR, I *was* two weeks ago... ;-)